operator/pkg/gateway-api/gatewayclass_status.go

```diff
@@ -4,24 +4,59 @@
 package gateway_api
 
 import (
+	"cmp"
+	"slices"
 	"time"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
+	"sigs.k8s.io/gateway-api/pkg/features"
 )
 
 const (
 	gatewayClassAcceptedMessage    = "Valid GatewayClass"
 	gatewayClassNotAcceptedMessage = "Invalid GatewayClass"
 )
 
+var supportedFeatures = features.AllFeatures
+
+var gatewayClassSupportedFeatures = getSupportedFeatures()
+
+var exemptFeatures = []features.SupportedFeature{
+	"GatewayStaticAddresses",
+	"HTTPRouteParentRefPort",
+	"MeshConsumerRoute",
+}
```

Comment:
just a small question, do we still need to specify the exempt feature in GHA workflow (e.g.  conformance-gateway-api.yaml)? or will the test framework read from gateway class status?

Reply:
We still havent added the change for the framework to read from GWC. We need to graduate this feature before we do it.

Reply:
ah thanks for sharing, and thanks for your work in upstream :medal_military: